applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -112,6 +112,10 @@ config BT_VCS
 	bool
 	default y
 
```

Comment:
So you dont have to set BT_PAC_SRC as default y?

Reply:
This is only changing the default, I think I we were to set CONFIG_BT_PACS_SRC_CONTEXT=0x05 then we would have an issue. Also, it doesn't look like we are actually using BT_PACS_SRC_CONTEXT yet, so I don't see the need to change the default

Reply:
I tried to build for CIS yesterday and then I got "Undefined reference CONFIG_BT_PACS_SRC_CONTEXT", even if it is default 0x03ff if BT_PAC_SRC=y

Reply:
**Same text as I wrote on Teams:**
Maybe I dont understand this correctly, but isnt it strange that we get compile errors caused by not finding BT_PACS_SRC_CONTEXT when we dont have any audio source in the device?



And it is kind of strange that defining new default values in Kconfig.defaults works, since CONFIG_BT_PAC_SRC is still not set (=n)